src/managers/gatewayCache.ts

```diff
@@ -14,14 +14,13 @@ export class GatewayCache {
     if (cacheName !== undefined) this.cacheName = cacheName
   }
 
-  async get(key: string): Promise<undefined | any> {
+  async get<T>(key: string): Promise<undefined | T> {
     const result = await this.client.cache.get(this.cacheName, key)
-    return result
+    return result as T
   }
 
-  async set(key: string, value: any): Promise<any> {
-    const result = await this.client.cache.set(this.cacheName, key, value)
```

Comment:
this looks like it could break some implementations depending on what `cache.set` returned previously

Reply:
it always returned undefined anyway

Reply:
oh yeah then it is fine.